flow/Knobs.cpp

```diff
@@ -312,6 +312,7 @@ void FlowKnobs::initialize(Randomize randomize, IsSimulated isSimulated) {
 	init( ENCRYPT_KEY_HEALTH_CHECK_INTERVAL,                    10 );
 	if ( randomize && BUGGIFY) { ENCRYPT_KEY_HEALTH_CHECK_INTERVAL = deterministicRandom()->randomInt(10, 60); }
 	init( EKP_HEALTH_CHECK_REQUEST_TIMEOUT,                    10.0);
+	init( ENCRYPT_KEY_CACHE_ENABLE_DETAIL_LOGGING,           false ); if( randomize && BUGGIFY ) ENCRYPT_KEY_CACHE_ENABLE_DETAIL_LOGGING = true;
```

Comment:
nit: do we want to retire `ENABLE_ENCRYPTION_CPU_TIME_LOGGING` knob? seems like now we have two knobs for controlling the profiling...